techlibs/gowin/synth_gowin.cc

```diff
@@ -112,30 +111,28 @@ struct SynthGowinPass : public ScriptPass
 		clear_flags();
 
 		size_t argidx;
-		for (argidx = 1; argidx < args.size(); argidx++)
-		{
-			if (args[argidx] == "-top" && argidx+1 < args.size()) {
+		for (argidx = 1; argidx < args.size(); argidx++) {
+			if (args[argidx] == "-top" && argidx + 1 < args.size()) {
 				top_opt = "-top " + args[++argidx];
 				continue;
 			}
-			if (args[argidx] == "-vout" && argidx+1 < args.size()) {
+			if (args[argidx] == "-vout" && argidx + 1 < args.size()) {
 				vout_file = args[++argidx];
 				continue;
 			}
-			if (args[argidx] == "-json" && argidx+1 < args.size()) {
+			if (args[argidx] == "-json" && argidx + 1 < args.size()) {
 				json_file = args[++argidx];
 				nobram = true;
 				nolutram = true;
-				nowidelut = true;
```

Comment:
Took me a while to find the actual key change between all the formatting changes haha